spec/system/view_data_in_s3_spec.rb

```diff
@@ -83,6 +83,20 @@
           expect(page).to have_content file1.filename
           expect(page).to have_content file2.filename
         end
+
+        context "when user is a super super_admin_user" do
```

Comment:
```suggestion
        context "when user is a super_admin_user" do
```

Probably accidental repetition? Though if there are super-super users, that would be good to know.